### ISSUE TITLE
Version for Blender 2.79.x (UPBGE 0.2.x)

### ISSUE THREAD

**Issue by Symbian9** [hlorus/geometry_sketcher#3]
Backport add-on for latest Blender 2.79.x (UPBGE 0.2.x)

- https://download.blender.org/release/Blender2.79/latest
- https://github.com/UPBGE/upbge/releases/tag/v0.2.5b

**Comment by hlorus** [comment]
Hey there,
I'm not sure what changes would be required to support older blender versions. While i think it's possible somehow i don't think this should be a priority right now  as the project is still in an experimental state.

**Comment by Symbian9** [comment]
> this should be a priority right now

No, I not ask to prioritize this task, guess its long term feature request.

Maybe I myself would do this work (in partial or in full).

**Comment by moueza** [comment]
LTS like Blender 2.93 or 3.2 yes for less work

**Comment by amrsoll** [comment]
It seems like the latest UPBGE Is Blender 2.8.x compatible, probably Blender LTS versions would be a good target after the 1.0 release, but I think it would require someone with the need for that to be the maintainer of that feature.
